app/utils/getUUID.ts

```diff
@@ -0,0 +1,9 @@
+import { v4 as uuidv4 } from 'uuid';
+
+//이용자의 uuid를 받아옵니다.
+const getUUID = () => {
+  const getUserUUID = (): string => uuidv4();
+  return getUserUUID;
+};
```

Comment:
요기 혹시 이렇게 한 이유가 있을까요? 이렇게 단순화할 수 있을 것 같아서요!
```suggestion
const getUUID = () => uuidv4();
```

Reply:
이 부분은 저의 화살표 함수 활용 미숙이네요...
이제 보니 왜 이랬나 싶군요
바로 반영하겠습니다~